### DISCUSSION TITLE
Is it possible to change the accent colour from grey to another colour?

### DISCUSSION THREAD

**Discussion by MyGuyCai** [rdbende/Azure-ttk-theme#32] [Question] [answered]
Hi, absolutely love the theme but would love to be able to change the accent colour from grey to another colour.  
Is this possible through code or do I have to change the png files?

**Comment by MyGuyCai** [comment]
Just adding that I have tried changing the style background colour with:  
ttk.Style().configure("TButton", background="red")

**Comment by rdbende** [comment, marked as answer]
Since every look is drawn from the png images, it's not possible to change it by a `Style.configure` call.
You need to change the images either dinamically at runtime, or statically.

**Reply by prakharjhudele** [reply]
Hi, @rdbende thanks for this awesome theme. Just wanted to make you aware of this UI issue. I have raised this on SO. But I think posting it here and bringing it to your notice will be helpful for the community. 
[UI background issue](https://stackoverflow.com/questions/72490123/using-ttk-theme-azure-dark-and-changing-background-color-of-ttk-combobox-widget).
Also, the combobox widget when disbaled has no effect. It might be because it is set to `disabled $I(box-basic)` just like list `$I(box-basic)`
